.changeset/cyan-mice-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
```

Comment:
```suggestion
"stylelint": minor
```

To align with our semver policy.

Reply:
Oh, it looks like I misunderstood the policy. False negatives are included in **minor** updates, right?

For example, https://github.com/stylelint/stylelint/releases/tag/15.10.3 includes a false negative fix released as a **patch**. Was this incorrect?

Reply:
> False negatives are included in minor updates, right?

Yes, "a bug fix in a rule that results in Stylelint reporting more errors".

> For example, https://github.com/stylelint/stylelint/releases/tag/15.10.3 includes a false negative fix released as a patch. Was this incorrect?

It was incorrect, but it's okay.

Reply:
Ah, I see it. I'll care about the policy more. 👍🏼